File: eBolnica/UI/ebolnica_mobile/lib/models/dnevnI_raspored_response.dart
import 'package:ebolnica_mobile/models/termin_model.dart';
import 'package:ebolnica_mobile/models/operacija_model.dart';

class DnevniRasporedResponse {
  int doktorId;
  String datum;
  List<Termin> termini;
  List<Operacija> operacije;

  DnevniRasporedResponse({
    required this.doktorId,
    required this.datum,
    required this.termini,
    required this.operacije,
  });

  factory DnevniRasporedResponse.fromJson(Map<String, dynamic> json) {
    return DnevniRasporedResponse(
      doktorId: json['doktorId'],
      datum: json['datum'],
      termini: (json['termini'] as List)
          .map((item) => Termin.fromJson(item))
          .toList(),
      operacije: (json['operacije'] as List)
          .map((item) => Operacija.fromJson(item))
          .toList(),
    );
  }
}

File: eBolnica/eBolnica.Model/Response/DnevniRasporedResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eBolnica.Model.Response
{
    public class DnevniRasporedResponse
    {
        public int DoktorId { get; set; }
        public DateTime Datum { get; set; }
        public List<Model.Models.Termin> Termini { get; set; } = null!;
        public List<Model.Models.Operacija> Operacije { get; set; } = null!;


    }
}
